server/ai/configuration.go

```diff
@@ -49,10 +57,30 @@ type BotConfig struct {
 }
 
 func (c *BotConfig) IsValid() bool {
```

Comment:
It might be nice to have this return an error with a specific failure. (I bet Aider could make this change pretty quick)